test/infrastructure/docker/hack/verify-all.sh

```diff
@@ -37,13 +37,6 @@ failed=()
 outputs=()
 
 # run all verify scripts, optionally skipping any of them
-if [[ "${VERIFY_DEPS:-true}" == "true" ]]; then
-  echo "[*] Verifying deps..."
-  out=$(hack/verify-deps.sh 2>&1)
-  failure $? "verify-deps.sh" "${out}"
-  cd "${REPO_PATH}" || exit
-fi
-
```

Comment:
Removes the secondary module verification in favor of the `make verify` target, which already includes the `verify-modules` target.

Reply:
Nice